golang/truv.go

```
@@ -106,16 +122,15 @@ func createUser() (string, error) {
-	res, err := client.Do(request)
+	response, err := http.DefaultClient.Do(request)
 	if err != nil {
 		return "", err
 	}
 
-	defer res.Body.Close()
+	defer response.Body.Close()
 
 	user := UserResponse{}
-	err = json.NewDecoder(res.Body).Decode(&user)
+	err = json.NewDecoder(response.Body).Decode(&user)
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)